.github/workflows/build.yml

```diff
@@ -81,6 +81,11 @@ jobs:
       matrix:
         compiler: [msvc, clang-cl]
         arch: [i386, amd64]
+        toolset: ['14.2', '14.1'] # VS 2019, 2017
+        include:
+        - compiler: msvc
+          arch: i386 # Not compiling on amd64 prompt
```

Comment:
Why?

Reply:
Because it fails to compile 16 bit code (THAT AGAIN, YES) for freeldr.
Look here: https://github.com/zefklop/reactos/runs/2527158409?check_suite_focus=true#step:11:650